OSXCore/InputReceiver.swift

```diff
@@ -172,12 +172,7 @@ extension InputReceiver { // IMKServerInput
         dlog(DEBUG_LOGGING, "LOGGING::COMMIT::%lu:%lu:%@", range.location, range.length, commitString)
         // NSLog("range1 \(range)")글
         if range.length == 0 {
-            range = sender.selectedRange()
-        }
-        // NSLog("range2 \(range)")
-        if range.length == 0 {
-            // 일부 프로그램이 길이가 0인 치환을 잘 처리하지 못하므로 특별히 처리한다
-            range = NSRange(location: NSNotFound, length: 0)
+            range = NSRange(location: NSNotFound, length: NSNotFound)
```

Comment:
여기서 length가 0이면 혹시 문제가 생기나요? 이전에 웨일에서 이게 NSNotFound면 오동작하는 버그가 있었거든요.

Reply:
테스트에 사용한 케이스(Chromium 브라우저에서의 구글 스프레드 시트 및, 블럭 후 두벌식 숫자 입력)에서는 length가 0인 경우에도 모두 동일하게 동작합니다. sender.selectedRange()를 사용하지 않는게 더 중요한 것 같아요.

Reply:
0이어서 문제가 없으면, length를 0으로 주면 좋을거 같아요.

```suggestion
            range = NSRange(location: NSNotFound, length: 0)
```